### PR TITLE
Rename to 'The Hallow' internally.

### PR DESCRIPTION
**Pull request by vini2003**
Initial PR, on hold until currently open PRs are closed.

Missing fixes:

- ~~`README.md`~~
- `de_de.json`
- `en_au.json`
- `en_gb.json`
- `en_nz.json`
- `en_us.json`
- `es_es.json`
- `fr_fr.json`
- `pt_br.json`
- `zh_cn.json`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # The Hallow
-Fabric Community mod for Hacktoberfest! Formerly known as Spooky Time.
+Fabric Community mod for Hacktoberfest! Formerly known as The Hallow.
```
Comment (Shnupbups) [inline README.md]: no

Comment (Shrimpstress) [reply README.md]: nay

Comment (vini2003) [reply README.md]: In case you didn't notice it:

    Missing fixes:
    - `README.md`

Comment (Shrimpstress) [reply README.md]: To be fair, I assumed this was in relation to before any changes were made at all, not changes made by this PR.

Comment (vini2003) [reply README.md]: Please review the new `README.md`.

**Comment by vini2003** [comment]
I'm going to fucking murder someone.

**Pull request merged**